crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -434,14 +441,42 @@ where
             Event::ChannelClosed {
                 channel_id,
                 reason,
-                user_channel_id: _,
+                user_channel_id,
             } => {
-                let channel = hex::encode(channel_id);
+                let channel_id = hex::encode(channel_id);
+                let user_channel_id = Uuid::from_u128(user_channel_id).to_string();
                 tracing::info!(
-                    %channel,
+                    %user_channel_id,
+                    %channel_id,
                     ?reason,
                     "\nChannel closed",
                 );
+
+                if let Some(mut channel) = self.storage.get_channel(&user_channel_id)? {
+                    if !vec![ChannelState::Pending, ChannelState::Open]
+                        .contains(&channel.channel_state)
+                    {
```

Comment:
🔧 I think something like `if matches!(x, Closed | ForceClosedRemote | ForceClosedLocal)` is your friend here.

You could even encapsulate that in a `Channel::is_closed` method, to call `if channel.is_closed()` here.